src/ecosystem/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Ecosystem"
-layout: base.njk
+layout: base-with-child-links.njk
```

Comment:
I wonder if there's a way we can autodetect this?  It's a little undesirable to have the frontmatter growing too many magic values for enabling features that we intend to be very normal.  And I don't think we'd ever want to turn this off.

Reply:
Yep, very good point, we're now autodetecting this 🥳